Needed for encoding large images with mencoder or grabbing large image from a v4l src

--- /tmp/swscale_internal.h	2010-10-05 09:15:42.000000000 +0200
+++ trunk/libswscale/swscale_internal.h	2010-10-05 09:16:07.000000000 +0200
@@ -33,11 +33,7 @@
 
 #define MAX_FILTER_SIZE 256
 
-#if ARCH_X86
 #define VOFW 5120
-#else
-#define VOFW 2048 // faster on PPC and not tested on others
-#endif
 
 #define VOF  (VOFW*2)
 
